test/test_openapi.py

```diff
@@ -11,28 +11,30 @@
 class TestGetAppSpecs(AzulTestCase):
 
     def test_unannotated(self):
-        # TODO: Once all endpoints are documented, this test should assert failure
+        # TODO: (jesse) Once all endpoints are documented, this test should assert failure
+        # (https://github.com/DataBiosphere/azul/issues/1450)
         app = self._mock_app_object()
         self._mock_app_route(app, '/foo')
-        app.get_all_specs()
+        undocumented = app.routes.keys() - app.path_specs.keys()
+        self.assertFalse(0 and undocumented)
```

Comment:
`@skip("https://github.com/DataBiosphere/azul/issues/1450")` before the test is what we typically use to disable tests until resolution of a ticket lands.

```suggestion
        self.assertEqual({}, undocumented)
```